src/sentry/integrations/github/tasks/codecov_account_link.py

```diff
@@ -31,6 +34,32 @@
 def codecov_account_link(
     integration_id: int,
     organization_id: int,
+) -> None:
+    link_codecov_account(integration_id, organization_id)
+
+
+@instrumented_task(
+    name="sentry.integrations.github.tasks.backfill_codecov_account_link",
+    queue="integrations.control.throttled",
+    max_retries=3,
+    silo_mode=SiloMode.CONTROL,
+    taskworker_config=TaskworkerConfig(
+        namespace=integrations_control_throttled_tasks,
+        retry=Retry(times=3),
+        processing_deadline_duration=60,
```

Comment:
These can be parameters to `instrumented_task()` now and you can skip all the celery parameters like `queue` and `max_retries`